ModuleData/tow_wanderertemplates.xml

```diff
@@ -2,19 +2,15 @@
 <NPCCharacters>
 	<NPCCharacter id="tow_wanderer_empire_0" name="{=!}{HERO.FIRSTNAME}, Bright Wizard" voice="curt" age="55" default_group="Infantry" is_template="true" is_hero="false" culture="Culture.empire" occupation="Wanderer">
```

Comment:
why is_hero =false for all templates? I thought from my side those characters are heroes.

Reply:
it turns out that they are actually not. Heros are only the lords afaik.

Reply:
it has some implications for the system I am currently working on. So far I count heroes as companions, and only heroes are able to gain or store Winds of magic. I don't know how important that flag is , but it needs to be find out what a companion truely is.

Reply:
wanderers are not companions until you hire them. Wanderer != companion. But you are right, companions are also heros.